src/clients/api/queries/getAuthentication/index.ts

```diff
@@ -0,0 +1,17 @@
+import { restService } from 'utilities';
+
+const getAuthentication = async (accountAddress: string | undefined): Promise<boolean> => {
```

Comment:
Just for the sake of making the name extra clear, I'd got with something like: `getIsAddressAuthorized`. What do you think?